### PR TITLE
Regenerate for arclight 1.0.0.beta4

### LINKED ISSUE #290: Remove dependency on ruby-sass which has reached end of life
ruby-sass has reached end of life, see:

- https://github.com/sass/ruby-sass
- https://sass-lang.com/blog/ruby-sass-is-unsupported

This project still depends on sass-rails 5.1.0 which depends on ruby-sass: https://github.com/sul-dlss/stanford-arclight/blob/master/Gemfile.lock#L356

### LINKED ISSUE #310: Update to use Blacklight v8, Arclight v1 beta
- [x] Assess whether it would be easier to start over with a new rails app and the latest versions of Blacklight and Arclight (SUL theme not required)
- [x] Review any customization that might have been made and assess what should be carried over
- [ ] Load sample data provided by Dinah (when available)

### PR DESCRIPTION
**Pull request by jcoyne**
Fixes #310
Fixes #290

### CODE REVIEW

File: config/repositories.yml
Comment (corylown) [inline config/repositories.yml:1]: We might want to revert the changes to this file so we retain the list of SUL libraries/repositories in the app.

**Review by corylown — CHANGES_REQUESTED** [review]
I think we want to keep the repository.yml config with the SUL libraries listed. Otherwise, this looks like a good place to start.

Given the solr changes we will also probably need to update the solr configs and reindex the data for prod and demo. I'll write up issues for those tasks.

File: config/repositories.yml
Comment (jcoyne) [reply config/repositories.yml:1]: I've restored the original repository configuration.
